docs/source/configuring-slack-for-spidermon.rst

```diff
@@ -0,0 +1,31 @@
+.. _configuring-slack-bot:
+
+====================================
+Configure Slack bot for Spidermon
+====================================
+
+What are bots?
+==============
+
+A bot is a type of Slack App designed to interact with users via conversation.
+
+To work with `Slack Actions <https://spidermon.readthedocs.io/en/latest/actions.html#slack-action>`_, you will need a Slack bot which would send `notifications <https://spidermon.readthedocs.io/en/latest/getting-started.html#slack-notifications>`_ to your Slack workplace from Spidermon.
+
+Steps
+=====
+
+.. note:
+    You would need to be leader of Slack workplace for which you are trying to create a bot.
+
+1. `Create Slack bot <https://get.slack.help/hc/en-us/articles/115005265703-Create-a-bot-for-your-workspace>`_ by following the steps listed on the link.
```

Comment:
```suggestion
1. `Create a Slack bot <https://get.slack.help/hc/en-us/articles/115005265703-Create-a-bot-for-your-workspace>`_ .
```

I think the last part is implied. If you are told to do something, and ‘do something’ is a link, you should expect that link to help you ‘do something’.

Reply:
Was just being descriptive.